lambda/format-user-record/.eslintrc.json

```diff
@@ -0,0 +1,19 @@
+{
+    "parser": "@typescript-eslint/parser",
+    "extends": [
+        "airbnb-typescript-prettier",
+        "plugin:@typescript-eslint/recommended",
+        "prettier"
+    ],
+    "settings": {
+      "import/resolver": {
+            "typescript": {
+                "alwaysTryTypes": true,
+                "paths": "./tsconfig.json"
+            }
+        }
+    },
+    "rules": {
+        "no-console": "off"
+    }
+}
```

Comment:
No worries for now, but if you get a chance see if you can turn on "new line at end of file" in VS Code
([Link here](https://stackoverflow.com/questions/44704968/visual-studio-code-insert-newline-at-the-end-of-files)).

That way when you hit save we'll get the same thing auto formatted.

I can't remember why new lines at the end of files are so important, I just tend to do it because otherwise Github gives you that little 🛑 sign 😂

Reply:
Strange it is enabled on my VScode and it still doesn't insert a new line